pkg/vdbgen/options.go

```diff
@@ -33,4 +33,5 @@ type Options struct {
 	CACertName         string
 	Krb5Conf           string
 	Krb5Keytab         string
+	DepotVolume        string
```

Comment:
Can we update the vdb-gen e2e test by using this new parameter?

Reply:
I don't see this change. There should be an update to `tests/e2e-leg-4/vdb-gen/run-vdb-gen.sh`